mycroft/skills/settings.py

```diff
@@ -376,37 +348,30 @@ def _get_remote_settings(self):
         """Get the settings for this skill from the server
 
         Returns:
-            skill_settings (dict or None): returns a dict if matches
+            skill_settings (dict or None): returns a dict on success, else None
         """
         try:
             remote_settings = self.api.get_skill_settings()
         except Exception:
             LOG.exception('Failed to download remote settings from server.')
-            success = False
-        else:
-            self.remote_settings = remote_settings
-            success = True
+            remote_settings = None
 
-        return success
+        return remote_settings
 
-    def _emit_settings_change_events(self):
-        for skill_gid, remote_settings in self.remote_settings.items():
+    def _emit_settings_change_events(self, remote_settings):
+        """Emit changed settings events for each affected skill."""
+        for skill_gid, skill_settings in remote_settings.items():
             settings_changed = False
             try:
-                previous_settings = self.last_download_result[skill_gid]
-            except KeyError:
-                if remote_settings:
-                    settings_changed = True
+                previous_settings = self.last_download_result.get(skill_gid)
             except Exception:
                 LOG.exception('error occurred handling setting change events')
             else:
-                if previous_settings != remote_settings:
+                if previous_settings != skill_settings:
                     settings_changed = True
             if settings_changed:
                 log_msg = 'Emitting skill.settings.change event for skill {} '
                 LOG.info(log_msg.format(skill_gid))
-                message = Message(
-                    'mycroft.skills.settings.changed',
-                    data={skill_gid: remote_settings}
-                )
+                message = Message('mycroft.skills.settings.changed',
```

Comment:
This change would be overwritten by Black.  You should just run Black on this module, as with any module we check in.

Reply:
Ok good to know, I'll revert this change.

Reply:
I'll leave complete blacking out this time since we should probably discuss which settings Black should use before starting to move from pycodestyle approved to black.

Reply:
The beauty of Black is that there really are no settings.  It is overly opinionated so that we don't have to be.  I don't agree with all the things Black does, but I put consistently styled code over my objections.